src/nat/middleware/red_teaming_middleware.py

```
@@ -0,0 +1,545 @@
+        elif self._target_field_resolution_strategy == "first":
+            return [matches[0]]
+        elif self._target_field_resolution_strategy == "last":
+            return [matches[-1]]
+        elif self._target_field_resolution_strategy == "all":
+            return matches
+        else:
+            raise ValueError(f"Unknown target_field_resolution_strategy: {self._target_field_resolution_strategy}")
+
+    def _apply_payload_to_complex_type(self, value: list | dict | BaseModel) -> list | dict | BaseModel:
+        if self._target_field is None:
+            raise ValueError(f"When attempting to apply payload to complex type, target_field is required")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)